config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
I agree about the moving average.
And yes, I was referring to regionIndex name. I don't mind changing it, but we should do use the same everywhere (I think it would only be for sst and ohc timeseries).